src/building/how-to-build-and-run.md

```diff
@@ -82,6 +79,33 @@ incremental = true
 verbose-tests = true
 ```
 
+Moreover, there are a couple of other configs to consider, depending on whether
+you are planning to work on something LLVM-related.
+
+If you are planning to work on LLVM-related things, you may want to enable
+assertions for improved debugging. This comes at a cost in compile time:
+
+```toml
+[llvm]
+# Indicates whether the LLVM assertions are enabled or not.
+assertions = true
+```
+
+If you are _not_ planning to work on LLVM-related things, you can avoid
+building LLVM altogether by using a system LLVM:
```

Comment:
I also wonder how good this suggestion is. If I do some changes let say on MIR and then try to run ui tests which compile and compare the output, couldn't using a different and unpatched LLVM version give problems?.

Reply:
Isn't MIR above the LLVM level? Why would it be affected by the llvm version?

Reply:
Maybe my example wasn't clear, I've just used MIR as an example my question is more general. By using some random LLVM system version that may not match the LLVM rustc version and do not have the rustc patches, are we sure rustc tests are going to pass? in particular ui tests.

Reply:
I think the UI tests should always pass, right? Most of them are long before LLVM runs. I would expect it to only matter for codegen. Is there a particular example you are thinking of?